cmd/arc_summary/arc_summary.py

```diff
@@ -241,8 +241,7 @@ def get_arc_summary(Kstat):
             'num': fBytes(mfu_size),
         }
 
-    elif arc_size < target_size:
-        mfu_size = (target_size - mru_size)
+    else:
```

Comment:
I'd argue that what we really want to report here is the mru/fmu percentages independent of the arc size.  In which case the conditional can be removed and this can be calculated as.

```
mru_per = fPerc(mru_size, mru_size + mfu_size)
mfu_per = fPerc(mfu_size, mru_size + mfu_size)
```